_pages/en/saas-and-cloud-strategy.md

```diff
@@ -0,0 +1,230 @@
+---
+layout: default
+title: SaaS vs Cloud Strategy
+lang: en
+ref: documents
+status: posted
+permalink: /saas-vs-cloud.html
+---
+
+## Context
```

Comment:
Split in two sections to differentiate the insanity from reasons:

1- Context

2- Purpose
(Note how I removed the word "attempts" :)

This document is to assist in: 
A) clarifying the terms used in the question (context)
B) guide decisions made with relations to cloud-based technogies

-- it looks here that you make a difference between Cloud and SaaS. Of so, the add a 

C)  guide decisions made with relations to SaaS

C/D) Define the strategy clarifying where each technology should be used to assist the organization in better delivering on its mandate.